config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml

```diff
@@ -48,13 +57,21 @@ periodics:
       - >-
         ./test/build.sh install-lib &&
         make test
+      resources:
+        limits:
+          cpu: 2
+          memory: 4Gi
+        requests:
+          cpu: 2
+          memory: 4Gi
```

Comment:
I kept the req., limit the same. Prow configures request as:
```yaml
resources:                                                                   
  requests:                                                                  
    cpu: 250m                                                                
    memory: 1Gi        
```

Ping me if you want me to change `request` to prow's `request`.